src/zimscraperlib/download.py

```diff
@@ -6,7 +6,7 @@
 import pathlib
 import subprocess
 from concurrent.futures import Future, ThreadPoolExecutor
-from typing import Dict, Optional, Union
+from typing import Dict, Optional, Tuple, Union
```

Comment:
I believe the _latest_ recommendation (since 3.9) is to use `tuple` instead of that imported `Tuple` from `typing`.
https://docs.python.org/3/library/typing.html#annotating-tuples

Reply:
See https://peps.python.org/pep-0585/#implementation as well

Reply:
You are right.
Since we are still supporting Python 3.7+, we still have to import `annotations` from `__future__`, this will be unnecessary once we will support only 3.9+.
I changed only my code, while this could be done on a more larger basis, I opened a ticket to made this significant change : https://github.com/openzim/python-scraperlib/issues/106.